### PR TITLE
Added the multiplication function

### PR DESCRIPTION
**Pull request by RValner**
Could somebody review the changes?

### CODE REVIEW

File: code_0.cpp [hunk 1]
```diff
@@ -2,3 +2,8 @@ int add(int a, int b)
 {
   return a+b;
 }
+
+float multiply(int a, int b)
```
Comment (RVTestUser) [inline code_0.cpp]: Change int arguments to float

**Review by RVTestUser — CHANGES_REQUESTED** [review]
See the line comment below:

File: code_0.cpp [hunk 2]
```diff
@@ -3,7 +3,7 @@ int add(int a, int b)
   return a+b;
 }
 
-float multiply(int a, int b)
+float multiply(float a, float b)
```
Comment (RVTestUser) [inline code_0.cpp]: Could you template this function?

**Review by RVTestUser — CHANGES_REQUESTED** [review]
See the comment

File: code_0.cpp (hunk 2 above)
Comment (RValner) [reply code_0.cpp]: Done, is it any good?

Comment (RVTestUser) [reply code_0.cpp]: Perfect, accepting the pull request!

**Review by RVTestUser — APPROVED** [review]

**Pull request merged**